src/views/v2/rsvp/attendees/attendee/rsvp-name.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Block: RSVP
+ * Attendees - Attendee RSVP Name
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/attendees/attendee/rsvp-name.php
```

Comment:
I believe for clarity this should be `rsvp-title` or `rsvp` (if we follow what we have for tickets), I see that we're using RSVP name elsewhere though. But I initially thought this was the attendee name. I know that this is subjective, I'm thinking about avoiding confusion for implementors.